stable/yugaware/values.yaml

```diff
@@ -84,6 +84,14 @@ yugaware:
     annotations: {}
     labels: {}
     terminationGracePeriodSeconds: 30
+    probes: # HTTP healthcheck probes configuration
+      enabled: false # add or not livenessProbe and startupProbe to the container
+      startupTimeSec:
+        min: 20 # minimum expected time for yugaware to start
+        max: 600 # maximum expected time for yugaware to start, k8s will restart pod on timeout
+      livenessProbe:
+        periodSeconds: 30 # query period
+        failureThreshold: 3 # number of failed checks to restart pod
```

Comment:
can we bump this up in default to something like 10 failed probes? 
YBM can set a more agressive threshold if needed?